### DISCUSSION TITLE
"shedskin: command not found" after installing on windows

### DISCUSSION THREAD

**Discussion by wbkdef** [shedskin/shedskin#468] [Q&A] [answered]
I've been going through the documentation:
https://shedskin.readthedocs.io/en/latest/documentation.html#windows

I have cmake installed (I can verify this in a developer command prompt).

I installed conan per the instructions: pip install ‘conan==1.62.0’

However this doesn't make a shedskin command available in the terminal (I've been using git bash).  There is now a conan command available, however.

Note, I'm using Windows 11, Python 3.12.

I also found "Contact: shedskin-discuss@googlegroups.com" at the bottom of this page, but wasn't able to send a message to it...
https://shedskin.github.io/

**Comment by humhue** [comment, marked as answer]
Did you install shedskin?
You can do so by cloning the repo, cd into it, then
pip install .

**Reply by wbkdef** [reply]
Thank-you!
